config/module/test-fixtures/basic-parent-providers/a/a.tf

```diff
@@ -0,0 +1,13 @@
+provider "top" {}
+
+provider "bottom" {
+    alias = "foo"
+    value = "from bottom"
+}
+
+module "b" {
+    source = "../c"
```

Comment:
Heh... it took me a moment here to get my head around the fact that this module is called `b` but its dir is called `c`. Not a big deal if there's a good reason for it, but it might be an aid to understanding to match these, otherwise.

Reply:
it came with the test fixture I copied ;) I'll make a note to change it up here for clarity.